src/typescript/sdk/src/indexer-v2/types/json-types.ts

```diff
@@ -364,6 +365,7 @@ export type DatabaseJsonType = {
       close_price_q64: Uint64String;
     }
   >;
+  [DatabaseRpc.RandomNames]: { emojis: string };
```

Comment:
Is there a more specific format to expect? It looks like it might be in the form of hex with an `x` prepended? It'd be nice to have it here just so it's clear what to expect when calling this function

You can do something like `x${string}`